packages/ibm-products/src/components/Datagrid/Datagrid/DatagridSelectAll.js

```diff
@@ -64,7 +65,15 @@ const SelectAll = (datagridState) => {
       <TableSelectAll
         {...selectProps}
         name={`${tableId}-select-all-checkbox-name`}
-        onSelect={onChange}
+        onSelect={(event) => {
```

Comment:
super minor, but can you move this out into a `onSelectHandler` or something? just to get those ugly if /else out of the jsx 😅